### DISCUSSION TITLE
From display name in email

### DISCUSSION THREAD

**Discussion by kdubba** [O365/python-o365#1066] [Q&A]
Hi all,
I am sending emails using 0365 and want to change the sender name. I am trying to do this as follows, but it doesn't seem to be working:
```
account = Account(
        credentials,
        auth_flow_type='credentials',
        tenant_id=CONFIG.O365_TENANT_ID,
        token_backend=tk,
    )
mailbox = account.mailbox(
    from_email
)
m = mailbox.new_message()
m.sender.address = from_email
m.sender.name = sender_name
m.to.add(to_email)
m.subject = subject
m.body = body
m.send() 
```

Any help would be appreciated.

Regards,
Krish

**Comment by alejcas** [comment]
m.sender = "account@example.com"

I think you need some kind of permission to be able to send from the address

**Comment by kdubba** [comment]
I do have permissions and I can send the email from the sender's email address. I just did not include the credentials part of the code in my question. I am just not able to set the name. Currently, it just takes it from MS settings, but I want to dynamically set the name because we send emails on behalf of some orgs, and we want to convey that in the name "our org on behalf of another org".

**Reply by alejcas** [reply]
I don't think this is something that is implemented but I suppose it should be easy to do
